samples/misc/NodeServicesExamples/NodeServicesExamples.csproj

```diff
@@ -0,0 +1,33 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
+    <AssemblyName>NodeServicesExamples</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>NodeServicesExamples</PackageId>
+    <RuntimeFrameworkVersion>1.1.0</RuntimeFrameworkVersion>
```

Comment:
This is an unsupported mix of runtime version and TFM. Change the framework to `netcoreapp1.1` and remove this property.

This is true of several other projects in the PR.